packages/story-editor/src/app/media/uploadQueue/index.ts

```diff
@@ -423,9 +472,12 @@ function useMediaUploadQueue() {
   );
 
   const uploadImage = useCallback(
-    async (item) => {
+    async (item: QueueItem) => {
       const { id, file, additionalData = {} } = item;
```

Comment:
`additionalData` is typed as required, so this `= {}` will never be used and can be removed.